mails/themes/classic/core/cheque.html.twig

```diff
@@ -19,9 +19,13 @@
         <td width="10">&nbsp;</td>
         <td>
           <font size="2" face="{{ languageDefaultFont }}Open-sans, sans-serif" color="#555454">
-            <p data-html-only="1" style="border-bottom:1px solid #D6D4D4;">
-              {{ 'Order {order_name}'|trans({}, 'Emails.Body', locale) }}&nbsp;-&nbsp;{{ 'Awaiting check payment'|trans({}, 'Emails.Body', locale)|raw }}
-            </p>
+            {% if templateType == 'html' %}
+
+              <p style="border-bottom:1px solid #D6D4D4;">
+                {{ 'Order {order_name}'|trans({}, 'Emails.Body', locale) }}&nbsp;-&nbsp;{{ 'Awaiting check payment'|trans({}, 'Emails.Body', locale)|raw }}
+              </p>
+            
+{% endif %}
```

Comment:
indent

Reply:
These templates are generated with a tool https://github.com/jolelievre/ps_convert_templates

So manually changing the indentation wold create an offset with the tool's output
And adding a proper indentation management in the tool seems overrated as this theme is the classic one which should not be used much in the future

Reply:
Le fu!

Reply:
What? "Le fu!" ?

Reply:
https://www.google.fr/search?q=le+fu+meme&source=lnms&tbm=isch&sa=X&ved=0ahUKEwin--e_gfDhAhX2BGMBHXRHAt4Q_AUIDigB&biw=1920&bih=937 😄